pygments/lexers/promql.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+"""
+    pygments.lexers.promql
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for Prometheus Query Language.
+
+    :copyright: Copyright 2006-2019 by the Pygments team, see AUTHORS.
```

Comment:
-2020 :)

Reply:
Yes, I hesitate to use 2020 because this [check script](https://github.com/pygments/pygments/blob/e59cdb5d26079a2fb35434dd7461db672955c2f1/scripts/check_sources.py#L37-L38) will fail because it looks for
```
r'^    :copyright: Copyright 2006-2019 by '
```
but I'll update my file to 2020.

Reply:
Ah, I need to fix that script then :) Thanks for letting me know!